editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/globalconfiguration/beans/BeanConfigUtil.java

```diff
@@ -53,19 +66,83 @@
  */
 public class BeanConfigUtil {
 
+	/*
+	 * This code reused from org.fusesource.ide.camel.editor.properties.creators.AbstractClassBasedParameterUICreator in the createBrowseButton method
```

Comment:
might worth extracting it in a dedicated class to reuse it, what do you think? Not too much differences?